assets/css/components/social-media.scss

```
@@ -0,0 +1,22 @@
+.social-media {
+  margin: 0 rem(8px);
+  color: $color-white-170;
+  @extend %font-ui-text-sm-oxygen;
+
+  a {
+    position: static;
+    color: $color-white-140;
+  }
+
+  .footer & {
+    margin: rem(32px) 0;
+  }
+
+  .icon {
+    margin-right: rem(5px);
+  }
+
+  #twitter-link {
+    margin-right: rem(15px);

```

```suggestion
    margin-right: rem(16px);
```
Remember, we work in multiples in 8 unless specifically told otherwise.